### PR TITLE
Splash screen

### PR DESCRIPTION
**Pull request by harshvardhan-takawale**
Splash Screen for app introduction

Please ensure that your pull request meets the following requirements - thanks !

* Follows our existing [codestyle](https://github.com/k9mail/k-9/wiki/CodeStyle).
* Does not break any unit tests.
* Contains a reference to the issue that it fixes.
* For cosmetic changes add one or multiple images, if possible.

### CODE REVIEW

**Comment by philipwhiuk** [comment]
Thanks for contributing! :+1: I've taken a fairly detailed review of the changes. Some of the stuff is build dependencies - we try not to change these when doing other stuff - it's helpful if those are in a separate commit. None of them seem vital. Also we manage the BuildTools version in a property rather than in each Gradle file.

For the actual code itself, it's mainly a question of ensuring we don't add untranslatable strings.

I've not viewed the actual screen yet - though the colours seem to be purple/pink. That matches the debug icon, but not the actual icon - so maybe we want separate colours for debug and for live. (can do this by having separate resource files for debug which override the live ones).

**Comment by harshvardhan-takawale** [comment]
Thank you so much for such detailed review. I will make all the changes you mention today itself. Can you too please build and view it on an emulator or phone and give me more review points, it worked fine on my device.

**Comment by philipwhiuk** [comment]
The build fails because Travis only picks up the old build-tools - it's defined in `travis.yml`

It worked on my phone - it's a bit purple... 😮 

Once you've updated the review I'll see if there's anything else 😃

**Comment by harshvardhan-takawale** [comment]
I'm done with the updates and had sent the new PR in the morning itself, but the travis-ci test still failed.

File: plugins/HoloColorPicker/build.gradle [hunk 1]
```diff
@@ -16,4 +16,5 @@ android {
     lintOptions {
         abortOnError false
     }
+    buildToolsVersion '26.0.2'
```
Comment (philipwhiuk) [inline plugins/HoloColorPicker/build.gradle]: You're still changing the buildToolsVersion here. This is causing the build failure.

Comment (harshvardhan-takawale) [reply plugins/HoloColorPicker/build.gradle]: Oh no, I clicked the update project button once and it seems to have changed the build version everywhere. Do I need to change this and send the PR again?

**Comment by harshvardhan-takawale** [comment]
Passed the checks! Thank you so much for your help! Also, can you let me know whether anything is being done so that the app follows the material design guidelines? I wish to work towards it. If you can tell me I am willing to work on it.

**Comment by philipwhiuk** [comment]
@daquexian has started some work on the AccountSetup stuff. Various other people have produced concepts and stuff - no other PRs though.

**Comment by harshvardhan-takawale** [comment]
Fixes issue #2984

**Comment by cketti** [comment]
Thanks for taking the time to work on a feature for K-9 Mail.

However, as mentioned in #2984 I don't want to change the "welcome" screen right now. Getting the improved account setup merged is far more important.
Once that's done we can talk about adding a tutorial. But I strongly believe we should start with design mockups instead of an implementation. That allows for much faster iterations to arrive at something we're comfortable adding to the app.

**Pull request closed without merging**